streaming/src/main/scala/org/apache/gearpump/streaming/appmaster/TaskSchedulerImpl.scala

```diff
@@ -174,5 +175,5 @@ class TaskSchedulerImpl(appId : Int, config: Config)  extends TaskScheduler {
 }
 
 object TaskSchedulerImpl {
-  case class TaskLaunchData(taskId: TaskId, taskDescription : TaskDescription, dag : DAG)
+  case class TaskLaunchData(taskId: TaskId, taskDescription : TaskDescription, subscriptions: List[Subscriber])
```

Comment:
Shall we add an application field here to provide general info about an application, e.g. user name, credential, config...?